fs/span-manager/span_manager.go

```diff
@@ -92,12 +94,13 @@ func (s *span) validateStateTransition(newState spanState) error {
 }
 
 type SpanManager struct {
-	cache    cache.BlobCache
-	cacheOpt []cache.Option
-	index    *compression.GzipZinfo
-	r        *io.SectionReader // reader for contents of the spans managed by SpanManager
-	spans    []*span
-	ztoc     *ztoc.Ztoc
+	cache                    cache.BlobCache
+	cacheOpt                 []cache.Option
+	index                    *compression.GzipZinfo
+	r                        *io.SectionReader // reader for contents of the spans managed by SpanManager
+	spans                    []*span
+	ztoc                     *ztoc.Ztoc
+	maxSpanVerificationTries int
```

Comment:
we should keep the name consistent, in config it's `MaxSpanVerificationRetries`, in default it's `defaultRetries`